languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -82250,6 +82262,16 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
                     <token min="0" max="4" postag="A"/>
                     <token postag="(V ppa|J) m .*" postag_regexp="yes"/>
                 </antipattern>
+                <antipattern>
+                    <token postag="UNKNOWN"/>
```

Comment:
The only difference from the pattern is the UNKNOWN tag, with this change, we will prevent the rule from triggering when the gender of the token is also unknown.